Gateway/Startup.cs

```diff
@@ -3,15 +3,13 @@
 // --------------------------------------------------------------------------------------------
```

Comment:
FYI, I sent a request to our CELA team about these headers (do we need them for open source code?). I will either remove them all or add them in all files in a future PR depending of their reply.

Reply:
I received a reply from CELA: we need these headers, and there might actually be other things we have to do... I'll track this in another PR.

---
In reply to: [272772514](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/30#discussion_r272772514) [](ancestors = 272772514)